python/fusion_engine_client/analysis/analyzer.py

```diff
@@ -27,7 +27,17 @@
 from ..utils import trace
 from ..utils.argument_parser import ArgumentParser, TriStateBooleanAction, CSVAction
 from ..utils.log import locate_log, DEFAULT_LOG_BASE_DIR
+
+
 _logger = logging.getLogger('point_one.fusion_engine.analysis.analyzer')
+__orig_warning = _logger.warning
+def bold_warning(msg, *args, **kwargs):
```

Comment:
eventually these should probably be in a shared code location (in utils)

Reply:
Yeah, this should probably really be a formatter class in our argparse utils file. I'll see if I can do that quickly instead. Definitely agree that we want to make that file shared code eventually.

Reply:
Scratch that, confused myself. This should be part of our `trace.py` addons for the `logging` library.

Reply:
Added a `HighlightFormatter` helper class to both offset the text like this and colorize it too.